views/pages/searchProjectsByTag.ejs

```diff
@@ -0,0 +1,211 @@
+            <li><a href="/projects">Project search</a></li>
+            <li class="active">Tag: <%= tag.name %></li>
+        </ol>
+    </div>
+
+
+    <div class="row">
+        <div class="col-md-10 col-md-offset-1">
+
+            <% if (user.admin) { %>
+
+            <div id="custom-toolbar">
+                <div class="form-inline" role="form">
+                    <a href="/project/add" class="btn btn-success">
+                        <i class="glyphicon glyphicon-plus"></i>
+                        Add
+                    </a>
+                    <button id="editBtn" class="btn btn-info" disabled="">
+                        <i class="glyphicon glyphicon-edit"></i>
+                        Edit
+                    </button>
+                    <button id="deleteBtn" class="btn btn-danger" disabled="">
+                        <i class="glyphicon glyphicon-remove"></i>
+                        Delete
+                    </button>
+                </div>
+            </div>
+
+            <% } %>
+            <table id="results"></table>
+        </div>
+    </div>
+
+
+</div>
+
+<% include ../partials/footer.ejs %>
+
+<script src="/lib/bootstrap-table/bootstrap-table.min.js"></script>
+<script src="/stickyTablePageSize.js"></script>
+<script src="/tags.js"></script>
+<script>
+$(function () { // when the DOM is ready
+    $table = $('#results');
+
+    TechRadar.getAllTags(function(allTags, error) {
+        $table.bootstrapTable({
+            method: 'get',
+            url: '/api/projects/tag/<%= tag.id %>',
+            cache: false,
+            striped: true,
+            escape: true,
+            idField: 'id',
+            search: true,
+            sortName: 'name',
+            showColumns: true,
+            showExport: true,
+            showRefresh: true,
+            searchOnEnterKey: true,
+            showPaginationSwitch: true,
+            pagination: true,
+            pageSize: page_size, // declared and set in stickyTablePageSize.js 
+            // reach the columns nested in the response data
+            responseHandler: function(res) {
+                return res.result; 
+            },
+            columns: [
+            <% if(user.admin) { %>
+                {
+                    field: 'state',
+                    checkbox: 'true'
+                }, 
+            <% } %>
+            {
+                field: 'id',
+                title: 'ID',
+                visible: false
+            }, {
+                field: 'name',
+                title: 'Name',
+                sortable: true,
+                formatter: operateFormatter
+            }, {
+                field: 'description',
+                title: 'Description',
+                sortable: false
+            }, {
+                title: 'Tags',
+                formatter: tagsFormatter,
+                sortable: true
+            }]
+        });
+
+        
+        function tagsFormatter(value, row, index) {
```

Comment:
if this function is identical to the one on the other view and you have tag.js created it might be worth to extract it to there - no code duplication then